File: .github/tools-matrix/fixture/py/sample.py
"""Module docstring with non-ASCII: héllo → 日本."""
import os.path
from collections import OrderedDict as OD

GREETING = "héllo → 日本"


class Server:
    """A server."""

    port = 8080

    def start(self, name):
        """Start the server."""
        def inner():
            return helper(name)
        return inner()


def helper(name):
    return os.path.join(GREETING, name)


def test_helper():
    assert helper("x")

File: .github/tools-matrix/fixture/ts/src/sample.js
import { readFile as rf } from "fs";
const path = require("path");

// Non-ASCII before the declarations: héllo → 日本
export const greeting = "héllo → 日本";

/** A server class. */
export class Server {
  port = 8080;
  start(name) {
    const inner = () => helper(name);
    return inner();
  }
}

function helper(name) {
  return path.join(greeting, rf(name));
}

it("starts the server", () => new Server().start("x"));
export { helper };
